clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/AwsConfiguration.groovy

```diff
@@ -179,6 +180,7 @@ class AwsConfiguration {
     int maxSecurityGroups = 5
     ReconcileMode reconcileClassicLinkSecurityGroups = ReconcileMode.NONE
     List<String> reconcileClassicLinkAccounts = []
+    AmazonBlockDevice unknownInstanceTypeBlockDevice
```

Comment:
was debating that myself ... original intention was the preserve the previous behavior (_which only would have been encountered had someone hand modified their pipeline JSON or made API calls directly_).

I think a reasonable default is fair, I'll go with a 20GB volume.